### DISCUSSION TITLE
Bug when using CTRL ALT DEL at On-Screen-Keyboard on Lock Screen

### DISCUSSION THREAD

**Discussion by schoolandme** [rustdesk/rustdesk#5859] [Feature Request]
I facing a weird situation.....

In my company environment there is some production kiosk machine which is touchscreen and do not have keyboard and mouse. After I install rustdesk client to these machine, they are unable to CTRL ALT DEL using on-screen keyboard at lock screen. The on-screen keyboard is functional, I already double-checked it. is just when it is lock screen press CTRL ALT DEL nothing happen, it would not unlock. I have to plug in a keyboard or rustdesk inside to help them unlock. Is not very convenience as they are shutdown machine every night.

After that I went to check on other pc or laptop which already install rustdesk, found out that all having this issue, regardless windows 7 or windows 10.

My rustdesk installer is build by myself with my host and public key done config follow this instructions https://rustdesk.com/docs/en/dev/build/all/ 

Does anybody facing this issue before?

**Comment by rustdesk** [comment]
Interesting.
